app/controllers/todos_controller.rb

```diff
@@ -19,18 +18,11 @@ def create
   end
 
   def destroy
-    params[:id].split(',').each do |id|
-      User.find(TodoList.find(id).user_id).increment!(:todos_number, -1)
-      TodoList.find(id).destroy
-    end
+    TodoList.where(:id => params[:id].split(',')).destroy_all
```

Comment:
as I can remember we decided to use `ids` instead of `id` because the value is plural also don't need to split the array. It should look like the following code:
```TodoList.where(id: params[:id]).destroy_all```